docs/conf.py

```diff
@@ -28,10 +28,25 @@
 # it
 
 # Copy the docs from Cliquet inside these ones.
-destination = os.path.join(__HERE__, 'api', 'cliquet')
+# XXX how do we copy on 2.x ?
+destination = os.path.join(__HERE__, 'api', '1.x', 'cliquet')
```

Comment:
I was thinking that the package documentation should only show the API that is implemented.
Note that we are displaying all version of the documentation in ReadTheDocs, so even if a documentation is only showing a version of the API, it will still be possible to read all version of the API.